.jfrog.taskcat.yml

```diff
@@ -3,32 +3,32 @@ project:
   owner: quickstart-eng@amazon.com
   s3_object_acl: private
   s3_regional_buckets: True
-  s3_bucket: tcat-422383ecc658557f9a377abae675aac0  # commercial accounts (also for GovCloud)
+  s3_bucket: tcat-34c17b83dcbd547393a86d8053a8d298 # commercial accounts (also for GovCloud)
```

Comment:
Not specific to this PR. We need to move away from using Taskcat created bucket for testing. This is an known issue that CSO office has asked us to resolve for a while now. We should create a permanent bucket for testing purpose.